ts/src/config.ts

```diff
@@ -102,6 +102,9 @@ export interface Config extends AuthenticationConfig {
   // too large.
   // https://nodejs.org/dist/latest-v9.x/docs/api/timers.html#timers_settimeout_callback_delay_args.
   serverBackoffCapMillis?: number;
+
+  // Allows user to specify API URL other than cloudprofiler.googleapis.com/v2.
```

Comment:
Nit: "other than https://cloudprofiler.googleapis.com/v2" so that it's verbatim of what it is and can serve as a reference.

Reply:
Done.